File: Hero/ClientApp/src/app/models/subjectivity-configuration/SearchSubjectivitiesResult.ts
export class SearchSubjectivitiesResult {
    id: string;
    texts: SubjectivityText[];
    type: string;
    isAutoAttaching: boolean;
    daysToResolve: number;
}

export interface SubjectivityText {
    text: string;
    languageIsoCode: string;
}

File: Hero/Models/SubjectivityConfiguration/SearchSubjectivitiesResult.cs
using System;
using System.Collections.Generic;

namespace Hero.Models.SubjectivityConfiguration;

public class SearchSubjectivitiesResult
{
    public Guid Id { get; set; }
    public ICollection<SubjectivityText> Texts { get; set; }
    public string Type { get; set; }
    public bool IsAutoAttaching { get; set; }
    public int DaysToResolve { get; set; }
}